bootstrap.php

```diff
@@ -24,7 +24,7 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-$rootDir = getenv('_PS_ROOT_DIR_');
+$rootDir = defined('_PS_ROOT_DIR_') ? _PS_ROOT_DIR_ : false;
```

Comment:
```suggestion
$rootDir = defined('_PS_ROOT_DIR_') ? _PS_ROOT_DIR_ : getenv('_PS_ROOT_DIR_');
```
@ytilotti can you try with this ?
In some conditions, one may be empty, but not the other. So we ensure that you get the right value before fallbacking.

Reply:
`getenv('_PS_ROOT_DIR_')` can work? Because to get it will need to be set.

Reply:
Yes in some case. Anyway, it will return false if nothing found is found.
It seems to work well on my environment and on github actions. So I want to check if it's ok in your configuration too.

Reply:
Yes will be ok because I match the first part.